rmgpy/rmg/main.py

```diff
@@ -1193,7 +1327,6 @@ def readMeaningfulLineJava(self, f):
         return line
     
 ################################################################################
-
```

Comment:
I believe I added this because it made it look more clean, I think it was suppose to be there.